src/de/quinscape/Terminal.java

```diff
@@ -2,44 +2,66 @@
 
 import java.io.FileWriter;
 import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
 import java.util.Scanner;
 import java.util.UUID;
 
 public class Terminal {
-    private String fileNameGenerator() {
-        UUID code = UUID.randomUUID();
-        return code.toString().substring(0,8);
+
+    private FileWriter file;
+
+    public void appendToFile() throws IOException {
+        printInstructions();
+        createNewFile();
+        writeText();
     }
 
-    private FileWriter createNewFile() throws IOException{
-        return new FileWriter(fileNameGenerator() + ".txt");
+    public void appendToGivenFile(String filename) throws IOException {
```

Comment:
Benenne diese Methode doch besser so: 
```suggestion
    public void appendToExisting(String filename) throws IOException {
```

Die Idee ist: Wir nehmen an, das diese Datei existiert. Mit "given" drückst du ja nur aus, das hier ein code - parameter kommt. Das sagt aber nicht aus, was damit gemeint ist. (Soll eine Datei mit diesem Namen erstellt werden?)